src/lightning/__setup__.py

```diff
@@ -47,6 +47,9 @@ def _prepare_extras() -> Dict[str, Any]:
     extras["app-extra"] = extras["app-cloud"] + extras["app-ui"] + extras["app-components"]
     extras["app-all"] = extras["app-extra"]
     extras["app-dev"] = extras["app-all"] + extras["app-test"]
+    extras["data"] = extras.pop("data-extra")
+    extras["data-all"] = extras["data-examples"] + extras["data"]
+    extras["data-dev"] = extras["data-examples"] + extras["data-test"] + extras["data"]
```

Comment:
Why this? It's not consitent with all the other projects

```suggestion
    extras["data-all"] = extras["data-examples"] + extras["data-extra"]
    extras["data-dev"] = extras["data-all"] + extras["data-test"]
```

Reply:
because we won't have base requirements, so it's always going to be extra only. Otherwise we'll have dependency conflicts with oldest torch versions.